specification/batch/data-plane/Microsoft.Batch/stable/2023-11-01.18.0/examples/PoolAdd_SecurityProfile.json

```diff
@@ -0,0 +1,33 @@
+{
+  "parameters": {
+    "batchUrl": "account.region.batch.azure.com",
+    "api-version": "2023-11-01.18.0",
+    "client-request-id": "00000000-0000-0000-0000-000000000000",
+    "ocp-date": "Wed, 14 Jun 2023 06:39:01 GMT",
+    "pool": {
+      "id": "mypool001",
+      "vmSize": "STANDARD_DC2s_V2",
+      "virtualMachineConfiguration": {
+        "imageReference": {
+          "publisher": "Canonical",
+          "offer": "UbuntuServer",
+          "sku": "18_04-lts-gen2",
+          "version": "latest"
+        },
+        "nodeAgentSKUId": "batch.node.ubuntu 18.04",
+        "securityProfile": {
+          "securityType": "trustedLaunch",
+          "encryptionAtHost": true,
+          "uefiSettings": {
+            "secureBootEnabled": false,
+            "vTpmEnabled": null
```

Comment:
Discussed during the board review that this line will get removed in a follow-up PR.